api/server.ts

```diff
@@ -9,12 +9,13 @@ BigInt.prototype.toJSON = function () {
     return this.toString();
 };
 
-const server = new ApiSocketServer(config.API.SOCKET.PORT, API_SOCKET_SERVER_CONFIG);
-server.run();
+process.addListener('unhandledRejection', err => {
```

Comment:
how about to extract this to some exception holder directory or utils.

Reply:
i think it`s realy important moment for place it in base file